tsdb/chunks/head_chunks.go

```diff
@@ -425,8 +425,8 @@ func repairLastChunkFile(files map[int]string) (_ map[int]string, returnErr erro
 	return files, nil
 }
 
-// WriteChunk writes the chunk to the disk.
-// The returned chunk ref is the reference from where the chunk encoding starts for the chunk.
+// WriteChunk writes the chunk to disk.
+// The returned chunk ref is the reference for where the chunk encoding starts for the chunk.
```

Comment:
Shouldnt that be `from` and not `for`

Reply:
Hm, maybe on second thought "from where" makes sense in this context. I'll revert this change.